backend/src/main/java/com/hackathon/backend/service/FavoriteDishService.java

```diff
@@ -0,0 +1,21 @@
+package com.hackathon.backend.service;
+
+import com.hackathon.backend.model.FavoriteDishes;
+import com.hackathon.backend.repository.FavoriteDishRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FavoriteDishService {
+
+    private final FavoriteDishRepository favoriteDishRepository;
+
+    @Autowired
+    public FavoriteDishService(FavoriteDishRepository favoriteDishRepository) {
+        this.favoriteDishRepository = favoriteDishRepository;
+    }
+
+    public FavoriteDishes insertFavoriteDishes(FavoriteDishes favoriteDishes) {
```

Comment:
We used to get a dto at the controller level and pass it to the service level, where that dto has to be converted into a model entity. So can you create FavoriteDishesDto, pass it to FavoriteDishesService, convert it to FavoriteDishes and store it in favoriteDishRepository?